### DISCUSSION TITLE
Marathi language

### DISCUSSION THREAD

**Discussion by devsm9** [AnySoftKeyboard/AnySoftKeyboard#2961] [Q&A]
Is marathi language available in this keyboard ??

**Comment by elektrawizard** [comment]
No.
Only 42 languages are available.
https://anysoftkeyboard.github.io/languages/
But you could make it as ASK is open source.

**Reply by menny** [reply]
https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/PACKS.md
A bit more, but not all in the Play Store

**Comment by menny** [comment]
We don't, but if you're knowledgeable in Android development you can follow the instructions at https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/CONTRIBUTING.md
